dapr_agents/workflow/orchestrators/base.py

```diff
@@ -43,6 +43,10 @@ def model_post_init(self, __context: Any) -> None:
         # Register agent metadata
         self.register_agentic_system()
 
+        # Start the runtime if it's not already running
+        logger.info("Starting workflow runtime...")
```

Comment:
there is a log in start_runtime() so probably we can remove this one to avoid repeating it:

https://github.com/dapr/dapr-agents/blob/main/dapr_agents/workflow/base.py#L442
```suggestion
```